integrations/airflow/marquez_airflow/dag.py

```diff
@@ -55,19 +53,19 @@
 
 class DAG(airflow.models.DAG, LoggingMixin):
     def __init__(self, *args, **kwargs):
+        self.log.info("MARQUEZ DAG")
         super().__init__(*args, **kwargs)
 
     def create_dagrun(self, *args, **kwargs):
+        self.log.error("------------------------REGISTER_DAGRUN")
         # run Airflow's create_dagrun() first
         dagrun = super(DAG, self).create_dagrun(*args, **kwargs)
 
         create_dag_start_ms = self._now_ms()
         try:
-            _MARQUEZ.create_namespace()
             self._register_dagrun(
                 dagrun,
-                DagUtils.get_execution_date(**kwargs),
-                DagUtils.get_run_args(**kwargs)
```

Comment:
is this something that needs to be added as a facet?

Reply:
I think it can be - if we think that whether this dag run is externally triggered is a meaningful information. 
As it already is in the repo, I'd add it.